fastapi/routing.py

```diff
@@ -655,21 +660,37 @@ def decorator(func: DecoratedCallable) -> DecoratedCallable:
         return decorator
 
     def add_api_websocket_route(
-        self, path: str, endpoint: Callable[..., Any], name: Optional[str] = None
+        self,
+        path: str,
+        endpoint: Callable[..., Any],
+        *,
+        dependencies: Optional[Sequence[params.Depends]] = None,
+        name: Optional[str] = None,
```

Comment:
Moving the `name` parameter around is a breaking change (here and on the other places).

Reply:
:thinking:  It seemed right to make it a kw-only argument and move it after `dependencies` for consistency with `add_api_route`.

However you are right, this is not the point of this PR, so I'm reverting it